.github/renovate.json

```diff
@@ -72,6 +72,45 @@
       "matchPaths": ["apps/**"],
       "groupName": "Rust applications toolchains",
       "commitMessageTopic": "Rust toolchains for `/apps`"
+    },
+    {
```

Comment:
Don't they also need a `commitMessageTopic`? Otherwise, the PR name might be not as expected.

Reply:
I don't think it's required, I like to try it out without specifying it explicitly, otherwise this should be an easy follow-up fix.